std_msgs/Bool bool
---
geometry_msgs/PoseArray[] graspPoints
